src/commands/moderation/Cases.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  EmbedBuilder,
+  PermissionsBitField,
+  SlashCommandSubcommandBuilder,
+  type ChatInputCommandInteraction
+} from "discord.js";
+import { genColor } from "../../utils/colorGen";
+import { getModeration, listUserModeration } from "../../utils/database/moderation";
+import { errorEmbed } from "../../utils/embeds/errorEmbed";
+import { randomise } from "../../utils/randomise";
+
+export default class Cases {
+  data: SlashCommandSubcommandBuilder;
+  constructor() {
+    this.data = new SlashCommandSubcommandBuilder()
+      .setName("cases")
+      .setDescription("Moderation cases in a server.")
+      .addUserOption(user =>
+        user.setName("user").setDescription("The user that you want to see.").setRequired(true)
+      )
+      .addStringOption(string =>
+        string.setName("id").setDescription("The ID of a specific moderation case you want to see.")
+      );
+  }
+
+  async run(interaction: ChatInputCommandInteraction) {
+    const actionsEmojis: { [key: string]: string } = {
+      WARN: "⚠️",
+      MUTE: "🔇",
+      KICK: "📤",
+      BAN: "🔨"
+    };
```

Comment:
Maybe group these in a constants file ? Idk if they are used elsewhere

Reply:
they are not